.travis.yml

```diff
@@ -2,18 +2,19 @@ language: c
 sudo: required
 install:
   - wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-opam.sh
-  - wget https://raw.githubusercontent.com/simonjbeaumont/ocaml-travisci-skeleton/docgen/.travis-docgen.sh
+  - wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-docgen.sh
 script:
   - bash -ex .travis-opam.sh
-  - ( ( test $TRAVIS_PULL_REQUEST = "false" && test $TRAVIS_BRANCH = "master" && bash -ex .travis-docgen.sh ) || true )
+after_success:
+  - test "$GEN_DOC" = 1 && test $TRAVIS_PULL_REQUEST = "false" && test $TRAVIS_BRANCH = "master" && bash -ex .travis-docgen.sh
```

Comment:
While we can expect `TRAVIS_PULL_REQUEST` and `TRAVIS_BRANCH` to be defined, I would have quoted them as well. But this can go in without quotes.